src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -126,6 +121,7 @@ public ExtensionsRunner() throws IOException {
             .put(TransportSettings.PORT.getKey(), extensionSettings.getHostPort())
             .build();
         this.customSettings = Collections.emptyList();
+        this.transportActions = new TransportActions(Collections.EMPTY_MAP);
```

Comment:
Prefer `Collections.emptyMap()` here (which just wraps `EMPTY_MAP` with an inferred `<T>` cast) which is friendlier to code/IDEs and avoids downstream unchecked cast warnings.